### PR TITLE
Infra/keyboard fixes

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Keyboard components - clean unused code, remove unused dependency, clean demo screens, and rename actual components in use.

## Changelog
Keyboard components - clean unused code, remove unused dependency, clean demo screens, and rename actual components in use.

### CODE REVIEW

File: demo/src/screens/nativeComponentScreens/keyboardAccessory/KeyboardAccessoryViewScreen.js [hunk 1]
```diff
@@ -170,7 +172,7 @@ export default class KeyboardInputViewScreen extends PureComponent {
   };
 
   render() {
-    const {message, isModal} = this.props;
+    const {message/* , isModal */} = this.props;
```
Comment (M-i-k-e-l) [inline demo/src/screens/nativeComponentScreens/keyboardAccessory/KeyboardAccessoryViewScreen.js:175]: Why not remove the `isModal` altogether? Where should it come from? Or is it an example?

File: lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m [hunk 2]
```diff
@@ -505,30 +505,11 @@ - (void) rctContentDidAppearNotification:(NSNotification*)notification
...
-        if ([presentedViewController isKindOfClass:[UINavigationController class]]) {
-            UINavigationController *presented = (UINavigationController*)presentedViewController;
-            if (presented.visibleViewController) {
-                return YES;
-            }
-        }
-    }
-    return NO;
-}
-
```
Comment (M-i-k-e-l) [inline lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m:525]: Do you know what was that for?

File: demo/src/screens/nativeComponentScreens/keyboardAccessory/KeyboardAccessoryViewScreen.js (hunk 1 above)
Comment (Inbal-Tish) [reply demo/src/screens/nativeComponentScreens/keyboardAccessory/KeyboardAccessoryViewScreen.js:175]: It is an example. I want to keep it

File: lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m (hunk 2 above)
Comment (Inbal-Tish) [reply lib/ios/reactnativeuilib/keyboardtrackingview/KeyboardTrackingViewTempManager.m:525]: Yes. I wrote it and forgot to remove it after I moved to another solution

File: demo/src/screens/nativeComponentScreens/keyboardAccessory/KeyboardAccessoryViewScreen.js [hunk 3]
```diff
@@ -105,21 +105,23 @@ export default class KeyboardInputViewScreen extends PureComponent {
   renderKeyboardAccessoryViewContent = () => {
     return (
       <View style={styles.keyboardContainer} paddingV-s4>
-        <View row paddingH-s4>
-          <TextInput
-            style={styles.textInput}
+        <View bg-white row spread centerV paddingH-s5 paddingV-s3>
+          <TextField
```
[the remark this answers is not in the document]
Comment (Inbal-Tish) [reply demo/src/screens/nativeComponentScreens/keyboardAccessory/KeyboardAccessoryViewScreen.js:109]: Done

**Review by M-i-k-e-l — APPROVED** [review]

**Pull request merged**